pkg/ibgu/ibgu_test.go

```diff
@@ -15,6 +18,11 @@ const (
 	testIbguNamespace = "test-namespace"
 )
 
+var (
+	defaultIbguName   = "ibgu-test"
+	defaultIbguNsName = "test-ns"
+)
```

Comment:
can just use testIbguName and testIbguNamespace instead

Reply:
Done thanks